dissect/target/plugins/apps/remoteaccess/remoteaccess.py

```diff
@@ -2,14 +2,15 @@
 from dissect.target.helpers.record import create_extended_descriptor
 from dissect.target.plugin import NamespacePlugin
 
-RemoteAccessRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
-    "application/log/remoteaccess",
-    [
-        ("datetime", "ts"),
-        ("string", "tool"),
-        ("string", "description"),
-        ("path", "logfile"),
-    ],
+GENERIC_LOG_RECORD_FIELDS = [
+    ("datetime", "ts"),
+    ("string", "tool"),
```

Comment:
I think the recent changes with the tool name in the record name has made this field unnecessary?